test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,75 @@
+package seedu.addressbook.common;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.tag.Tag;
+import java.util.ArrayList;
+
+import java.util.List;
+
+public class UtilsTest {
+
+    private Utils utils;
+
+    @Before
+    public void setup() {
+        utils = new Utils();
+    }
+
+    @Test
+    public void EmptyInput_returnsTrue() {
+        List<Tag> emptyTags = new ArrayList<>() ;
+        emptyTags.clear();
+        assertTrue(utils.isAnyNull(emptyTags));
+        }
+    @Test
+    public void elementsAreNotUniqueInput_returnsFalse() throws IllegalValueException {
+        
+        Tag sameTag = new Tag ("friendly");
+        List<Tag> listOfSameTags = new ArrayList<>() ;
```

Comment:
You can see that some of the codes are reused, so you may wish to refactor the code a little more